SPECS/moby-containerd/moby-containerd.spec

```diff
@@ -1,51 +1,32 @@
 %global debug_package %{nil}
+%define upstream_name containerd
+%define commit_hash ad771115b82a70cfd8018d72ae489c707e63de16
 
 Summary: Industry-standard container runtime
-Name: moby-containerd
-Version: 1.5.9+azure
+Name: moby-%{upstream_name}
+Version: 1.6.0
 Release: 1%{?dist}
 License: ASL 2.0
 Group: Tools/Container
-
-# Git clone is a standard practice of producing source files for moby-* packages.
-# Please look at ./generate-sources.sh for generating source tar ball.
-
-%define vernum %(echo "%{version}" | cut -d+ -f1)
-Source0: https://github.com/containerd/containerd/archive/v%{vernum}.tar.gz#/%{name}-%{version}.tar.gz
-Source1: containerd.service
-Source2: containerd.toml
-Source3: NOTICE
-Source4: LICENSE
 URL: https://www.containerd.io
 Vendor: Microsoft Corporation
 Distribution: Mariner
 
+Source0: https://github.com/containerd/containerd/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
I've noticed in `Makefile.patch` that we're building with `-mod=vendor`, so I wanted to double-check: are the Go dependencies already pre-packed inside the source tarball downloaded from the upstream or did we have to download them separately and build our custom source tarball? In case it's the latter, let's also include instructions how to build the tarball. If it's the former - sorry for the trouble.;)